pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -483,7 +446,7 @@ func getDeployment(el *v1alpha1.EventListener) *appsv1.Deployment {
 	}
 }
 
-func getContainer(el *v1alpha1.EventListener) corev1.Container {
+func (r *Reconciler) getContainer(el *v1alpha1.EventListener) corev1.Container {
```

Comment:
Same as above, I think its a bit clearer to pass in the Config struct as a Param